src/Effects/Tremolo.js

```diff
@@ -0,0 +1,83 @@
+Pizzicato.Effects.Tremolo = function(options) {
+
+	this.options = {};
+	options = options || this.options;
+
+	var defaults = {
```

Comment:
one thing about this tremolo2 branch, is that the square and sawtooth shapes seem to click a lot. I'm not 100% sure why that is.